flight/Modules/UAVOFrSKYSensorHubBridge/UAVOFrSKYSensorHubBridge.c

```diff
@@ -256,6 +254,9 @@ static void uavoFrSKYSensorHubBridgeTask(void *parameters)
 	GPSPositionData gpsPosData;
 	BaroAltitudeData baroAltitude;
 	FlightStatusData flightStatus;
+
+	uint8_t *serial_buf = frsky_state->serial_buf;
```

Comment:
Is this just to create a shorthand variable? I don't immediately see what's wrong with ```frsky_state->serial_buf```, it's pretty compact like that. And ```PIOS_COM_SendBuffer(frsky_state->frsky_port, serial_buf, msg_length);``` already includes the use of ```frsky_state``` once so having it in the function a second time won't be awkward.